.github/workflows/pr-dev.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           password: ${{ env.DOCKERHUB_PWD }}
           buildargs: BUILD_TYPE
           dockerfile: Dockerfile
-          tags: "${{ steps.fetch-docker-vars.outputs.REPO_VERSION }}"
+          tags: "${{ env.REPO_VERSION }}"
```

Comment:
I just found the issue by looking at your PR, it should have been steps.fetch-vars.outputs instead of steps.fetch-DOCKER-vars.outputs

Reply:
I'll do a commit on your branch and thumbs it up